legend-engine-config/legend-engine-extensions-collection-generation/pom.xml

```diff
@@ -120,6 +120,11 @@
             <groupId>org.finos.legend.engine</groupId>
             <artifactId>legend-engine-xt-javaPlatformBinding-pure</artifactId>
         </dependency>
+        <dependency>
+            <groupId>org.finos.legend.engine</groupId>
```

Comment:
why is this needed ?

Reply:
This is to pass "public void testMetadataDeserializer()" at TestExtensions.java